### DISCUSSION TITLE
Joining new devices only VERY close to coordinator

### DISCUSSION THREAD

**Discussion by dakky** [Koenkk/zigbee2mqtt#15199] [Devices]
Hello. I have mostly Osram/LEDVANCED zigbee plugs. In order to join them. I have to bring them EXTRMELY close to to coordinator (about 10cm),

When trying to pair them farer away, the get detected, but pairing (interviewing) fails. Any Idea how to fix that or what might be wrong here?

Coordinator: CC2652RB stick (USB-A)

**Comment by ikarisan** [comment]
I am trying to join some LEDVANCE SMART+ panels (60x60cm and 120x30cm tunable white) but they are not even recognized by any of my routers/coordinators. Neither the coordinator (zStack30x, CC2538 + CC2592) nor all the other routers (IKEA Tradfri light bulbs, Girier TS0001..) in my network are recognizing the LEDVANCE devices. But my 8€ "Tuya TYZG1 Smart Gateway" can connect to all the lights within 30 seconds. :( 

After 10-20 retries I managed to join the 60x60 panel by holding it 4cm away from an IKEA LED1545G12 bulb. But the 120x30 panels cannot be joined to my network at all. :(
